homeassistant/components/fireservicerota/config_flow.py

```diff
@@ -5,7 +5,8 @@
 from homeassistant import config_entries
 from homeassistant.const import CONF_PASSWORD, CONF_TOKEN, CONF_URL, CONF_USERNAME
 
-from .const import DOMAIN, URL_LIST  # pylint: disable=unused-import
+# pylint: disable=relative-beyond-top-level
```

Comment:
Why do we need to disable this pylint error?

Reply:
It was the only pylint error in this code which is bogus, but I will remove it, no prob.